src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -308,7 +308,10 @@ private void OnExpanderButtonPreviewMouseMove(object sender, MouseEventArgs e)
                 return;
 
             var senderButton = e.OriginalSource as FrameworkElement;
-            dragDropHelper.HandleMouseMove(senderButton, e.GetPosition(null));
+            var searchElementVM = senderButton.DataContext as NodeSearchElementViewModel;
+            if (searchElementVM != null)
+                dragDropHelper.HandleMouseMove(senderButton, e.GetPosition(null)); 
+
```

Comment:
@aosyatnik - yes. Also, there is another simple fix. 
OnExpanderButtonMouseLeftButtonDown - comment if (searchElementVM != null). That case, drag occurs only for node elements (not for the top level).  I am fine with both fix'es.  If there is any special reason for this check  if (searchElementVM != null), then I will go with the next one (that is checking for null in OnExpanderButtonPreviewMouseMove).